report/src/main/resources/application.properties

```diff
@@ -37,6 +38,9 @@ spring.kafka.producer.batch-size=16384
 spring.kafka.producer.buffer-memory=33554432
 spring.kafka.producer.key-serializer=org.apache.kafka.common.serialization.StringSerializer
 spring.kafka.producer.value-serializer=org.springframework.kafka.support.serializer.JsonSerializer
+        
+spring.main.allow-bean-definition-overriding: true
```

Comment:
This is wrong, bean should not be allowed to be overridden. If it is being allowed that means code has an issue